test/trento/application/integration/discovery/protocol/enrich_register_application_instance_test.exs

```diff
@@ -31,6 +31,34 @@ defmodule Trento.EnrichRegisterApplicationInstanceTest do
              Enrichable.enrich(command, %{})
   end
 
+  test "should not return an enriched command if the database instance host has been deregistered" do
```

Comment:
nitpick: better say "should return a database not found error if....
so it's cleared what is doing in terms of domain